pkg/kubelet/volume_host.go

```diff
@@ -128,6 +128,16 @@ func (kvh *kubeletVolumeHost) GetPodsDir() string {
 	return kvh.kubelet.getPodsDir()
 }
 
+// GetHostIDsForPod if the pod uses user namespaces, takes the uid and gid
+// inside the container and returns the host UID and GID those are mapped to on
+// the host. If containerUID is nil, then it returns the host UID for UID 0
+// inside the container. If containerGID is nil, then it returns nil.
+// If the pod is not using user namespaces, as there is no mapping needed, the
+// same containerUID and containerGID params are returned.
+func (kvh *kubeletVolumeHost) GetHostIDsForPod(pod *v1.Pod, containerUID, containerGID *int64) (hostUID, hostGID *int64, err error) {
```

Comment:
it would be good to get someone from sig storage to weigh in here.

i have a bias to not change call paths unless the feature gate was on, so if this feature gate was off, it would never actually be called, or it would return an error if it was called.

cc @jsafrane @gnufied

Reply:
I think it should be okay as it is written right now. The argument for keeping feature gate check inside `getHostIDsForPod` is - it makes unit testing etc simpler and code inside operation_generator can always mock desired value if need be.

Reply:
Thanks!